src/services/ContributingDeveloperAuditService/scm/GitHub/api/GitHubApiClient.ts

```diff
@@ -0,0 +1,215 @@
+import axios, { AxiosInstance, AxiosResponse } from "axios";
+import { soosLogger } from "../../../../../logging/SOOSLogger";
+import { sleep } from "../../../../../utilities";
+
+interface IHttpRequestParameters {
+  baseUri: string;
+  githubPAT: string;
+}
+
+interface IHttpClientParameters extends IHttpRequestParameters {
+  apiClientName: string;
+}
+
+export interface GitHubOrganization {
+  login: string;
+}
+
+export interface GitHubRepository {
+  id: number;
+  name: string;
+  full_name: string;
+  private: boolean;
+  owner: GitHubOrganization;
+}
+
+export interface ContributingDeveloper {
```

Comment:
If we want to store the `scriptversion` as I suggested we might need to change this interface @SOOS-JJennings

Reply:
The version of the script (this script) that was run to generate the audit details?

Reply:
It will be the version of the new package (only if we are interested of keeping track of it)
https://github.com/soos-io/soos-scm-audit/pull/2

Reply:
makes sense - we can add a "auditScriptVersion" param - I'll add that to the BE ticket

Reply:
![image](https://github.com/soos-io/soos-api-client/assets/88039131/f25908c8-87ae-42d7-962a-ed4f225b8d23)